frontend/src/app/ruleset-creator/ruleset-creator.component.ts

```diff
@@ -1,18 +1,30 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, HostBinding } from '@angular/core';
 import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { RulesetCreateDto, RulesetsBackendService } from '@ovide/backend';
 
 import * as faker from 'faker';
 import { Router } from '@angular/router';
+import { trigger, transition, style, animate } from '@angular/animations';
 
 
 @Component({
   selector: 'ovide-ruleset-creator',
   templateUrl: './ruleset-creator.component.html',
-  styleUrls: ['./ruleset-creator.component.scss']
+  styleUrls: ['./ruleset-creator.component.scss'],
+  animations: [
+    trigger('fadeIn', [
+      transition(':enter', [
+        style({ transform: 'scale(0.9)', opacity: 0.2 }),
+        animate('.3s ease-out')
+      ])
+    ])
+  ]
 })
 export class RulesetCreatorComponent implements OnInit {
 
+
+  @HostBinding('[@slideInOut]')
```

Comment:
This line seems to cause problems and throws the following error:

> ERROR in `@HostBinding` parameter should be a property name, 'class.<name>', or 'attr.<name>'.

The initial animation does not disappear. I also tried a clean install.

Reply:
it shouldnt be there anyway i just forgot to remove it